src/main/java/org/pmiops/workbench/model/Cohort.java

```diff
@@ -0,0 +1,25 @@
+package org.pmiops.workbench.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class Cohort {
```

Comment:
Another thing to consider is defining models in swagger and using code generation. (I don't know how well this works in practice, but it could be worth a quick exploration.)